arbeitszeit_flask/templates/user/409.html

```diff
@@ -0,0 +1,9 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+<div class="section has-text-centered">
+    HTTP 409 NOT FOUND
```

Comment:
s/NOT FOUND/CONFLICT/  — Do we need to support this status code?  In what scenario?